benchmark-prefs.json

```diff
@@ -0,0 +1,9 @@
+{
+  "xpinstall.signatures.required": false,
+  "javascript.options.strict": false,
+  "extensions.@activity-streams.sdk.console.logLevel": "error",
+  "extensions.@activity-streams.performance.log": false,
+  "extensions.@activity-streams.previews.enabled": false,
+  "extensions.@activity-streams.telemetry": true,
+  "extensions.@activity-streams.telemetry.ping.endpoint": "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream"
+}
```

Comment:
Please also add

```
  "browser.crashReports.unsubmittedCheck.autoSubmit": true
```

so that we don't get those "Submit unsent crash reports?" bottom bars rendered, since they'll change the rendering timing.